utils/Formatter.tsx

```diff
@@ -47,12 +47,12 @@ export function convertTagToName(itemTag?: string): string {
     formatted = capitalizeWords(formatted)
 
     // special per item Formating
-    formatted = formatted?.replace('Pet Item', '')
+    formatted = formatted?.replaceAll('Pet Item', '')
     if (formatted?.startsWith('Pet')) {
-        formatted = formatted?.replace('Pet', '') + ' Pet'
+        formatted = formatted?.replaceAll('Pet', '') + ' Pet'
     }
     if (formatted?.startsWith('Ring')) {
-        formatted = formatted?.replace('Ring ', '') + ' Ring'
+        formatted = formatted?.replaceAll('Ring ', '') + ' Ring'
```

Comment:
These changes are not needed.
You only want to switch the first occurence of the words to the end. If it would occur at some other place you wouldnt want to remove it